src/solvers/dgmulti/types.jl

```diff
@@ -124,7 +215,51 @@ function DGMultiMesh(dg::DGMulti{NDIMS}; cells_per_dimension,
     @. vertex_coordinates[i] = 0.5 * (vertex_coordinates[i] + 1) * domain_lengths[i] + coordinates_min[i]
   end
 
-  return DGMultiMesh(vertex_coordinates, EToV, dg, is_on_boundary=is_on_boundary, periodicity=periodicity)
+  md = MeshData(vertex_coordinates, EToV, dg.basis)
+  if NDIMS==1
+    md = StartUpDG.make_periodic(md, periodicity...)
+  else
+    md = StartUpDG.make_periodic(md, periodicity)
+  end
+  boundary_faces = StartUpDG.tag_boundary_faces(md, is_on_boundary)
+  return DGMultiMesh{NDIMS, GeometricTermsType(Cartesian(), dg), typeof(md), typeof(boundary_faces)}(md, boundary_faces)
+end
+
+"""
+    DGMultiMesh(dg::DGMulti{NDIMS}, cells_per_dimension, mapping;
+                is_on_boundary=nothing,
+                periodicity=ntuple(_ -> false, NDIMS), kwargs...) where {NDIMS}
+
+Constructs a `Curved()` [`DGMultiMesh`](@ref) with element type `dg.basis.elementType`. The domain is
+the tensor product of the intervals `[coordinates_min[i], coordinates_max[i]]`.
```

Comment:
Where can I find `[coordinates_min[i], coordinates_max[i]]`?

Reply:
This is a mistake - thanks for the catch. Removing now